gnovm/tests/imports.go

```diff
@@ -58,9 +58,9 @@ const (
 	ImportModeNativePreferred
 )
 
-// ImportModeStdlibsOnly: use stdlibs/* only (except a few exceptions). for stdlibs/* and examples/* testing.
-// ImportModeStdlibsPreferred: use stdlibs/* if present, otherwise use native. for files/tests2/*.
-// ImportModeNativePreferred: do not use stdlibs/* if native registered. for files/tests/*.
+// ImportModeStdlibsOnly: use pkg/stdlibs/* only (except a few exceptions). for pkg/stdlibs/* and examples/* testing.
+// ImportModeStdlibsPreferred: use pkg/stdlibs/* if present, otherwise use native. for files/tests2/*.
+// ImportModeNativePreferred: do not use pkg/stdlibs/* if native registered. for files/tests/*.
```

Comment:
I'll (ab)use this line as it made me realise something in the structuring here:

1. I see in your branch on github that there are still two directories in the top-level `stdlibs` directory - I imagine this is not intentional?
2. I'm not sure I agree with moving stdlibs into a sub-package, because in many ways it is not a go package, and it is also very important & central to gnovm, don't you think? I would argue that its new place should be `gnovm/stdlibs`.

Reply:
1. Fixed, will be pushed soon.
2. It is indeed a Go package. I agree with you, let's move it to gnovm/stdlibs.